circuit/src/rns/secp256k1.rs

```diff
@@ -0,0 +1,222 @@
+use super::*;
+
+#[derive(Debug, Clone, PartialEq, Default)]
+/// We implement two structs one for the Secpk256k1 Base Field as the wrong field and one for the Secp256k1 Scalar Field as the wrong field. The native field is the BN256 scalar field.
+/// The reason for implementing both these structs is that ECDSA verification contains operations in both fields.
+/// Struct for the Secp256k1 Base Field as the wrong field. https://github.com/privacy-scaling-explorations/halo2curves/blob/main/src/secp256k1/fp.rs
+///
+/// Secp25k1 aux points
+/// to_add.x: 0x25968a86095277f8a984c449dc3447d5b2007a27b9eece0db6fb9ae69217bae0
+/// to_add.y: 0xdb08482232d4e3cf46c81d9b71d2247b87c83eb80e46c645758b5bfd51a955d8
+/// to_sub.x: 0xf6530b63da8d89214d6a0cfe38f7294317ebe6f8cd408c9617a123c2b0a7b025
+/// to_sub.y: 0x0903073bbd64df08681cf59bf4689b77e18b198eb3833371e39cf322ff8f1de3
+pub struct Secp256k1BaseField4_68;
```

Comment:
```suggestion
pub struct Secp256k1_Base_4_68;
```

Reply:
The linter does not allow me to do this because of the `#[deny(non_camel_case_types)]` constraint.